pandas/tests/frame/methods/test_diff.py

```diff
@@ -169,3 +169,28 @@ def test_diff_sparse(self):
         )
 
         tm.assert_frame_equal(result, expected)
+
+    def test_diff_integer_na(self):
```

Comment:
the OP also was testing for int64, can you parameterize on that as well (result is of course float64 in that case)

Reply:
Numpy ints are already tested higher up this file, so I think it is fine to focus here on nullable int

Reply:
ok that's fine, let's co-locate the tests then